jobs/view_generated_bios.py

```
@@ -0,0 +1,41 @@
+
+
+def main():
+    print("=" * 80)
+    print("GENERATED BIOS VIEWER")
+    print("=" * 80)
+
+    # Read all generated bios
+    generated_bios = read_all_generated_bios()
+    
+    # Calculate statistics
+    num_handles = len(set(bio.handle for bio in generated_bios))
```

Ruff (0.14.5)

Unnecessary generator (rewrite as a set comprehension)

Rewrite as a set comprehension

(C401)